### PR TITLE
Frontend -  board displaying, setting units

### PR DESCRIPTION
**Pull request by arkadiuss**

### CODE REVIEW

File: frontend/src/components/Store.tsx [hunk 1]
```diff
@@ -1,23 +1,82 @@
...
+  units,
+  unitsPlacement,
+  buyUnit,
+  placeUnit,
+}: Props) => {
+  const range = (n: number): number[] => {
+    return Array.from(Array(n).keys());
+  };
+  const [selectedUnit, selectUnit] = useState<string>("");
+  const board = range(BOARD_HEIGHT).map(_ => range(BOARD_WIDTH).map(_ => ""));
```
Comment (zemiret) [inline frontend/src/components/Store.tsx:24]: There should be a separate component for board, it shall not be placed in store. But I'll take that over.

**Comment by zemiret** [comment]
Great! I'm very pleased it works. I'll take over this branch :P and add a few fixes in communicator + update it to my styling work, and then style it.

File: frontend/src/components/Store.tsx (hunk 1 above)
Comment (arkadiuss) [reply frontend/src/components/Store.tsx:24]: Good point!

File: frontend/src/components/Backpack.tsx [hunk 2]
```diff
@@ -5,15 +5,25 @@ import UnitCmp from "./Unit";
 
 export interface Props {
   units: Unit[];
+  selectUnit: (unit: Unit) => void;
+  selectedUnit: Unit | null;
```
Comment (grzegorz-wcislo) [inline frontend/src/components/Backpack.tsx:9]: You can use `selectedUnit?: Unit`

File: frontend/src/components/GamePage.tsx [hunk 3]
```diff
@@ -16,12 +18,16 @@ interface Props {
   round: number;
   phaseEndsAt: number;
   units: Unit[];
+  selectedUnit: Unit | null;
```
Comment (grzegorz-wcislo) [inline frontend/src/components/GamePage.tsx:21]: Same as above

**Pull request merged**